listener.py

```diff
@@ -42,6 +46,7 @@ def client_listener(self):
                 #Kicks off election to higher hosts
                 contestableHosts = list(filter(lambda x: int(x[0].split('.')[3]) > int(self.thisNode.getIPAddress.split('.')[3]), self.thisNode.getCurrentHosts))
 
+                #Sending a request for election when more than one host apply for master
```

Comment:
This comment is very close, but the contestableHosts are a list of hosts that have got a higher last octet than the current node. So maybe you want to change your comment to read something like:

#Lists of hosts eligible for election. This node will send election message to nodes with higher last IP octet only